zebedee-cms/src/main/java/com/github/onsdigital/zebedee/kafka/KafkaClientImpl.java

```diff
@@ -36,7 +36,8 @@ public KafkaClientImpl(String kafkaAddr, String topic) {
      */
     @Override
     public Future<RecordMetadata> produceContentPublished(String url, String dataType, String collectionID) {
-        ContentPublished value = new ContentPublished(url,dataType,collectionID);
+        String uri = url.substring(0, url.length() - "/data.json".length());
```

Comment:
Will the url always have a suffix of `/data.json`?

If not then wouldn't this remove the length of `/data.json` and could result in throwing an error/exception?

Reply:
This will remove the suffix of /data.json, if exists, and it should be fine.